### ISSUE TITLE
Testing #203089 : Alerts are read but text in editor is no longer read

### ISSUE THREAD

**Issue by aiday-mar** [microsoft/vscode#203113]
Testing #203089

While testing I noticed that when I moved my cursor on a line with an error, the word `error` was read, but the contents of that line were no longer read. I was wondering if this is intended?

I noticed that sometimes the beginning of the text on the line is read, before it breaks off and reads the alerts.

https://github.com/microsoft/vscode/assets/61460952/b9446f47-1d9d-4d5d-bd5f-7c9029622dae

**Comment by aiday-mar** [comment]
I noticed that sometimes when I put my cursor on a line with a breakpoint, it does not read the word `Breakpoint`. Is the alert only supposed to sound at specific moments? Ie, for example the first time the cursor is placed on an active line with a breakpoint?

https://github.com/microsoft/vscode/assets/61460952/4be409bc-0c12-4560-951f-506fbd594984

**Comment by meganrogge** [comment]
@aiday-mar are you in screen reader optimized mode?

**Comment by meganrogge** [comment]
@aiday-mar, alerts interrupt whatever else might be read by design. 

And yes, it looks like in your case, you're using mouse clicks vs arrow keys to navigate between the lines. We listen to arrow keys, thus it's expected that you wouldn't hear the cues.

**Comment by meganrogge** [comment]
Additionally, I have checked with our power SR users and the alerts are working as expected in the editor.

**Comment by meganrogge** [comment]
I am going to open an issue to get more feedback on this.

**Comment by aiday-mar** [comment]
Hi @meganrogge, thank you for the clarification, that makes sense.

**Comment by meganrogge** [comment]
We are going to use `status` instead here because editor content is the priority.
